templates/zabbix_proxy.conf.erb

```diff
@@ -356,6 +356,17 @@ UnavailableDelay=<%= @unavaliabledelay %>
 #
 UnreachableDelay=<%= @unreachabedelay %>
 
+## Option: StartODBCPollers
```

Comment:
I'd place the `if @zabbix_version.to_f >= 6.0` before this line, as the comment doesn't make sense on 5.x either.

Reply:
Done.